src/main/scala/org/hammerlab/magic/test/listener/TestSparkListener.scala

```diff
@@ -158,7 +158,12 @@ object TestSparkListener {
 
   var instance: TestSparkListener = _
 
-  def apply(): TestSparkListener = instance
+  def apply(): TestSparkListener = {
+    if (instance == null) {
```

Comment:
This is done for cases when listener calls `clear()`, but application is not initialised yet. Similar change is made in another PR, will resolve conflicts once one of them is merged.